### PR TITLE
Add blog link to top nav

### PR DESCRIPTION
**Pull request by jakeatdocforce**
## High Level Overview of Change

Add link to blog at top nav. 

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/utils/config.ts [hunk 1]
```diff
@@ -37,6 +37,19 @@ export const getCharityAppURL = (): string => {
...
+      return 'https://blog.payid.org/'
+    case 'stage':
+      return 'https://blog.payid.org'
+    case 'prod':
+      return 'https://blog.payid.org/'
+    default:
+      return 'https://blog.payid.org/'
+  }
+}
+
```
Comment (Stormtv) [inline src/utils/config.ts:52]: If these are all going to be the same we should just hard code this in the nav

**Review by Stormtv — CHANGES_REQUESTED** [review]
@jakeatdocforce 

The screen is not properly responsive at the different breakpoints due to the new menu item being added. Also I don't think we need getBlogLocation() if there is only one blog environment.

File: src/utils/config.ts (hunk 1 above)
Comment (jakeatdocforce) [reply src/utils/config.ts:52]: Thought about hardcoding but it's possible to have 'dev' for local.  Still remove them for now?

Comment (jakeatdocforce) [reply src/utils/config.ts]: Hardcoded for now and fix for spacing

**Review by Stormtv — APPROVED** [review]
Looks great, approved.

**Pull request merged**